packages/headless/src/test/mock-commerce-facet-value.ts

```diff
@@ -19,6 +20,20 @@ export function buildMockCommerceRegularFacetValue(
   };
 }
 
+export function buildMockCommerceLocationFacetValue(
+  config: Partial<LocationFacetValue> = {}
+): LocationFacetValue {
+  return {
+    value: '',
+    state: 'idle',
+    numberOfResults: 0,
+    isAutoSelected: false,
```

Comment:
You need to check what you want, but except value and state other fields are ignored and you could remove them. I understand that this is because of the shared FacetValue base class. And would need significant refactoring.

Reply:
Will also tackle in https://github.com/coveo/ui-kit/pull/4574!